.github/workflows/ci.yml

```diff
@@ -130,12 +130,23 @@ jobs:
         python .\aws-c-s3\build\deps\aws-c-common\scripts\appverifier_ctest.py --build_directory .\aws-c-s3\build\aws-c-s3
 
   osx:
-    runs-on: macos-11 # latest
+    runs-on: macos-12 # latest
     steps:
     - name: Checkout Sources
       uses: actions/checkout@v2
```

Comment:
minor: can you also update actions/checkout to v3? to get rid of deprecated warnings with node 12